clang/include/clang/Basic/LangOptions.def

```diff
@@ -515,7 +515,7 @@ VALUE_LANGOPT(FuchsiaAPILevel, 32, 0, "Fuchsia API level")
 // on large _BitInts.
 BENIGN_VALUE_LANGOPT(MaxBitIntWidth, 32, 128, "Maximum width of a _BitInt")
 
-COMPATIBLE_LANGOPT(IncrementalExtensions, 1, 0, " True if we want to process statements"
+COMPATIBLE_LANGOPT(IncrementalExtensions, 1, 0, " True if we want to process statements "
```

Comment:
I'm not entirely sure where description of langoption goes, but I think a space in the beginning is not needed.
```suggestion
COMPATIBLE_LANGOPT(IncrementalExtensions, 1, 0, "True if we want to process statements "
```